Trans Awareness Month post (#4)
ByteOtter/journey: content/posts/trans_pride/index.md

we can all feel it. And when one of us thrives, we all do.

I know things might feel bleak right now. The world and society can feel like it's turning on us, and the weight of it can be a lot to carry.
But, friend, I want you to remember something else: **It is okay to not be okay**. You don't have to have everything figured out right now.
You don't have to carry the burden of the world on your shoulders. It's okay to have moments of doubt, of frustration, of pain.
But please remember: **You are loved so much**. You are seen. You are precious. *And yes, things will get better*.

As someone, who has only recently discovered her true self, I often have to remind myself of that too.
Sometimes, I sit in these holes of agony not understanding what is going on, fearing the future. But it's important to reach out for help. I'm never alone. **You are never alone**.
We are all here for each other, whether we've known each other for years or we've just met. There's a strength in our community.
And that strength doesn't disappear just because some orange and old people won an election. They are out of touch with reality.

> So, as we celebrate Trans Awareness Month together, take a moment to breathe. Take a sip of your hot chocolate or tea -- or whatever it is you are drinking -- and allow yourself

Reviewer: Maybe shorten it a bit?

```suggestion
So, as we celebrate Trans Awareness Month together, take a moment to breathe. Take a sip of your preferred beverage and allow yourself
```

Author: Yep I understand where you're coming from. Idk though. The post is supposed to feel like a friend is talking to the reader or sends a voice message and I think this style fits that goal quite well.
But I'll look if I can maybe shorten that while keeping the original feeling. Thanks for the review ^^
